app/javascript/components/Profile/EditPage.tsx

```diff
@@ -194,9 +196,9 @@ export const EditProfile = (props: Props) => {
 
   return (
     <SectionReducerContext.Provider value={reducer}>
-      <header className="grid grid-cols-1 gap-4 border-b border-border px-4 py-8">
+      <header className="relative grid grid-cols-1 gap-4 border-b border-border px-4 py-8">
```

Comment:
Since `SectionToolbar` uses `lg:absolute`, should be use `lg:relative`? Just wanna make sure nothing breaks because we decided to make this relative for all screens.
```suggestion
      <header className="lg:relative grid grid-cols-1 gap-4 border-b border-border px-4 py-8">
```